pkg/controller/direct/cloudbuild/workerpool_controller.go

```diff
@@ -142,6 +142,29 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 }
 
 func (m *model) AdapterForURL(ctx context.Context, url string) (directbase.Adapter, error) {
+	// Format: //cloudbuild.googleapis.com/projects/<project>/lcoations/<location>/workerPools/<id>
```

Comment:
can we build this from the existing `CloudBuildWorkerPoolIdentity`?

Reply:
Chatted offline, we're going to punt on this for a bit as return nil, nil is really important to the registry.Export func